### PR TITLE
Update array symbols in api.py

### LINKED ISSUE #4: В этой библиотеке нет поддержки  properties-attributes
Допустимые значения для массивов
'variants' : 'variant',
'variants-attributes' : 'variant',
'options' : 'option',
'option-names' : 'option-name',
'option-values' : 'option-value',
'orders' : 'order',
'order-lines-attributes': 'order-line-attributes',
'images' : 'image',

Как-нибудь можно добавить 'properties-attributes' : 'properties-attribute', в arrays = .. в insales/api.py?

+ Еще исправьте  проблему с кодировкой в setup.py
При установке через pip install pyinsales вылетает ошибка в этом файле.
замените  строчку readme_content на : 
readme_content = open("README.md",'r', encoding='utf8').read()

### PR DESCRIPTION
**Pull request by knopki**
There are many plural/singular forms for array types. So I created a script that parses api.insales.ru and creates a dictionary of that forms.

Closes #4

### CODE REVIEW

**Comment by knopki** [comment]
@nkrkv check this out!

File: insales/api.py [hunk 1]
```diff
@@ -8,14 +8,95 @@
...
+        'delivery-zones-attributes': 'delivery-zones-attribute',
+        'description-translations': 'description-translation',
+        'discount-codes': 'discount-code',
+        'discount-collections': 'discount-collections',
+        'discount-order-lines-ids': 'discount-order-lines-ids',
+        'discount-products-ids': 'discount-products-ids',
+        'discounts': 'discounts',
+        'discounts-attributes': 'discounts-attribute',
+        'domains': 'domain',
+        'errors': 'errors',
```
Comment (nkrkv) [inline insales/api.py]: This one looks suspicious

File: insales/api.py [hunk 2]
```diff
@@ -8,14 +8,95 @@
...
+        'field-options': 'field-option',
+        'field-options-attributes': 'field-options-attribute',
+        'field-values-attributes': 'field-values-attribute',
+        'fields': 'field',
+        'fields-values': 'fields-value',
+        'fields-values-attributes': 'fields-values-attribute',
+        'files': 'file',
+        'images': 'images',
```
Comment (nkrkv) [inline insales/api.py]: And this. Obviously breaks things

File: insales/api.py [hunk 3]
```diff
@@ -8,14 +8,95 @@
...
+        'js-tags': 'js-tag',
+        'locations': 'location',
+        'locations-attributes': 'locations-attribute',
+        'menu-items': 'menu-item',
+        'menus': 'menu',
+        'nil-classes': 'nil-classes',
```
Comment (nkrkv) [inline insales/api.py]: this

File: insales/api.py [hunk 4]
```diff
@@ -8,14 +8,95 @@
...
+        'objects': 'variant-field',
```
Comment (nkrkv) [inline insales/api.py]: Very suspicious

File: insales/api.py [hunk 5]
```diff
@@ -8,14 +8,95 @@
...
+        'options-attributes': 'options-attribute',
+        'order-changes': 'order-change',
+        'order-lines': 'order-line',
+        'order-lines-attributes': 'order-lines-attribute',
+        'orders': 'order',
+        'outlets': 'outlet',
+        'pages': 'page',
+        'payment-delivery-variants': 'payment-delivery-variant',
+        'payment-delivery-variants-attributes': 'payment-delivery-variants-attribute',
+        'payment-method': 'payment-method',
```
Comment (nkrkv) [inline insales/api.py]: Uh?

File: insales/api.py [hunk 6]
```diff
@@ -8,14 +8,95 @@
...
+        'supplementary-ids': 'supplementary-id',
+        'tags': 'tag',
+        'tariffs': 'tariff',
+        'tariffs-attributes': 'tariffs-attribute',
+        'title-translations': 'title-translation',
+        'variant-field-values': 'variant-field-value',
+        'variant-field-values-attributes': 'variant-field-values-attribute',
+        'variants': 'variant',
+        'variants-attributes': 'variants-attribute',
+        'warnings': 'warnings',
```
Comment (nkrkv) [inline insales/api.py]: :male_detective:

File: insales/api.py [hunk 7]
```diff
@@ -8,14 +8,95 @@
...
+        'menus': 'menu',
+        'nil-classes': 'nil-classes',
+        'objects': 'variant-field',
+        'option-names': 'option-name',
+        'option-values': 'option-value',
+        'options': 'option',
+        'options-attributes': 'options-attribute',
+        'order-changes': 'order-change',
+        'order-lines': 'order-line',
+        'order-lines-attributes': 'order-lines-attribute',
```
Comment (nkrkv) [inline insales/api.py]: Differs from original in line/lines plurality

File: insales/api.py (hunk 4 above)
Comment (knopki) [reply insales/api.py]: Hm, `objects` is a special case. So, I'm ignoring it.

File: insales/api.py (hunk 5 above)
Comment (knopki) [reply insales/api.py]: Only one singular form in docs. Ignoring.

File: insales/api.py (hunk 7 above)
Comment (knopki) [reply insales/api.py]: There is no `order-lines-attribute` in docs. Error?

Comment (nkrkv) [reply insales/api.py]: Well, api.insales.ru is right here

**Comment by knopki** [comment]
@nkrkv all fixed

**Pull request merged**